pom.xml

```diff
@@ -52,27 +55,25 @@
         <version>${junit.version}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+          <groupId>io.rest-assured</groupId>
+          <artifactId>rest-assured</artifactId>
+          <version>${rest-assured.version}</version>
+          <scope>test</scope>
```

Comment:
You already have test scope in the submodule, so this one is not necessary. Same for other test dependencies.

Also, personally I think test scope in dependencyManagement isn't a good idea, because then scope definition may be skipped in the submodule and make it confusing on a first look.

Reply:
I see your point and I must say I agree that it can be confusing

Reply:
Fixed